core/tests/protractor/explorationFeedback.js

```diff
@@ -47,16 +48,17 @@ describe('ExplorationFeedback', function() {
   it('adds feedback to an exploration', function() {
     var feedback = 'A good exploration. Would love to see a few more questions';
     var feedbackResponse = 'Thanks for the feedback';
+    var creatorDashboardPage = new CreatorDashboardPage();
 
     // Creator creates and publishes an exploration
     users.login('user1@ExplorationFeedback.com');
     workflow.createAndPublishExploration(EXPLORATION_TITLE,
                                          EXPLORATION_CATEGORY,
                                          EXPLORATION_OBJECTIVE,
                                          EXPLORATION_LANGUAGE);
-    browser.get(general.CREATOR_DASHBOARD_URL);
+    creatorDashboardPage.get();
     var numberOfFeedbackMessages = (
-      creatorDashboard.getNumberOfFeedbackMessages());
+      creatorDashboardPage.getNumberOfFeedbackMessages());
```

Comment:
Nit: can we set the following pattern instead, here and below:

    expect(
      creatorDashboardPage.getNumberOfFeedbackMessages()
    ).toEqual(0);

No need to declare extra variables if not needed.

Reply:
Done -- Agreed.